NuclearData/nuclearDataReg_mod.f90

```diff
@@ -13,6 +13,7 @@
 !! Available ND TYPES:
 !!   CE_NEUTRON
 !!   MG_NEUTRON
+!!   MG_PHOTON
```

Comment:
I would make this MG_IMC for now given the physics is quite different to what a more regular (linear) photon might be.